### ISSUE TITLE
invalid-first-character-of-tag-name error in {{ }}

### ISSUE THREAD

**Issue by xfanwu** [vuejs/eslint-plugin-vue#590]
**Tell us about your environment**

* **ESLint Version:**

v5.6.0

* **eslint-plugin-vue Version:**

v4.7.1

* **Node Version:**

v8.9.1

**Please show your full configuration:**

```javascript

module.exports = {
    root: true,
    parserOptions: {
      parser: 'babel-eslint'
    },
    env: {
      browser: true,
      node: true,
      es6: true,
      mocha: true
    },
    // https://github.com/vuejs/eslint-plugin-vue#priority-a-essential-error-prevention
    // consider switching to `plugin:vue/strongly-recommended` or `plugin:vue/recommended` for stricter rules.
    extends: [
      'standard',
      'eslint:recommended',
      'plugin:vue/essential',
      'plugin:vue/strongly-recommended'
    ],
    // required to lint *.vue files
    plugins: [
      'vue'
    ],
    // check if imports actually resolve
    settings: {
      'import/resolver': {
        webpack: {
          config: 'build/webpack.base.conf.js'
        }
      }
    },
    // add your custom rules here
    "rules": {
      "prefer-const": ["error", {
        "destructuring": "any",
        "ignoreReadBeforeAssign": true
      }],
      "comma-spacing": ["error", { "before": false, "after": true }],
      "no-console": ["error", { "allow": ["info", "error"] }],
      "quotes": [2, "single", "avoid-escape"],
      "semi": ["error", "never"],
      "no-var": 2,
      "no-unused-vars": ["error", {
        "varsIgnorePattern": "[iI]gnored",
        "args": "none",
        "caughtErrors": "all",
        "ignoreRestSiblings": true
      }],
      "camelcase": [1],
      "indent": ["error", 4],
      "comma-dangle": 1,
      "max-len": 1,
      "no-multi-str": 1,
      "global-require": 1,
      "no-mixed-operators": 1,
      "space-before-function-paren": ["error", {
        "anonymous": "always",
        "named": "always",
        "asyncArrow": "always"
      }],
      "no-trailing-spaces": ["error"],
      "object-curly-spacing": ["error", "always"],
      "keyword-spacing": ["error"],
      "key-spacing": ["error", { "beforeColon": false }],
      "padding-line-between-statements": [
        "error",
        { "blankLine": "always", "prev": "*", "next": "return" },
        { "blankLine": "always", "prev": "block-like", "next": "*" },
        { "blankLine": "always", "next": "block-like", "prev": "*" },
        { "blankLine": "always", "prev": ["const", "let", "var"], "next": "*" },
        { "blankLine": "any", "prev": ["const", "let", "var"], "next": ["const", "let", "var"] }
      ],
      // VUE rules:
      "import/no-duplicates": 1,
      "vue/no-reserved-keys": 0
    }
  }
  


```

**What did you do? Please include the actual source code causing the issue.**

```vue

<span>{{ content.replace(/<[^>]+>/g,"") }}</span>

```

**What did you expect to happen?**

There should not be an error for the RegExp code in `{{ }}`

**What actually happened? Please include the actual, raw output from ESLint.**

![image](https://user-images.githubusercontent.com/6542647/46192535-fb306e80-c32d-11e8-97b1-2f41a37c5202.png)

**Comment by armano2** [comment]
this issue is relaed to https://github.com/mysticatea/vue-eslint-parser
@mysticatea

**Comment by mysticatea** [comment]
Thank you for the report.

However, this is a duplicate of #370 and #474.
I recommend using computed properties or filters.
